scripts/asset_processor/video_metrics.py

```diff
@@ -468,14 +467,14 @@ def compute_metrics(self,
 
         rendition_metrics = {}
         # Some metrics only need the luminance channel
-        reference_frame_gray = reference_frame
-        rendition_frame_gray = rendition_frame
-        next_reference_frame_gray = next_reference_frame
-        next_rendition_frame_gray = next_rendition_frame
+        reference_frame_v = cv2.cvtColor(reference_frame, cv2.COLOR_BGR2HSV)[..., -1]
```

Comment:
Just to make sure that I understand correctly:

We're still computing most of the metrics using just the V component by converting the frame from BGR to HSV here, but we're computing certain metrics such as `temporal_histogram_distance` using all color channels from the BGR frame? And in order to accomplish this we pass the BGR frames to this VideoMetrics class so the VideoMetrics class can do the HSV conversion instead of doing the HSV conversion and then passing the HSV frames to the VideoMetrics class?

Reply:
Yes, that's right. Full image metric works in conjunction with single-channel metrics.